packages/v4/src/content/design-guidelines/components/progress/progress.md

```diff
@@ -3,102 +3,187 @@ id: Progress
 
-Use a progress bar in a wizard to demonstrate to the user where they are in the flow.
+<img src="./img/progress-bar-left-aligned-progress-value.png" alt="A visual example of a progress bar with left-aligned progress value text in line with the progress track." />
 
-## Styling
-Styling for the progress bar should follow accessibility guidelines. The styling demonstrated below should guide design decisions.
+Always set an “outside fixed width measure” for determinate progress bars. This fixed spacing should solve alignment use cases in which stacked progress bar values vary in length. 
 
-A blue progress bar represents a process in progress.
+## Placement
 
-![progressbar](./img/progress-bar-blue.png)
+<img src="./img/progress-bar-do.png" alt="A visual example of how to successfully place a progress bar in your designs, aligned with corresponding content." />
 
-A red progress bar represents a process that has failed, and the accompanying the times-circle icon demonstrates the error that has occurred in the process.
+<img src="./img/progress-bar-dont.png" alt="A visual example of how not to place a progress bar in your designs: Never isolate a progress bar above, below, or outside of its corresponding content view." />
 
-![progressbar](./img/progress-bar-error-message.png)
+Always place progress bars in alignment with their relevant context, such as the feature, function, or task they reflect. Keep progress bars within the user’s line of sight: Never isolate a progress bar at the top or bottom of a page, or outside of its relevant content view.
 
-A green progress bar represents the successful completion of a process, and the accompanying check-circle icon demonstrates the error that has occurred in the process.
+Use progress bars to communicate progress in a variety of contexts including:
 
-![progressbar](./img/progress-bar-success.png)
+* [In a table](#in-a-table)
+* [In a dashboard view](#in-a-dashboard-view)
+* [During a file download](#during-a-file-download)
+* [In a wizard](#in-a-wizard)
+
+### In a table
+
+Use a progress bar in a table to communicate the status of processes or tasks within it.
+
+Place a progress bar into a table just as you would other table content. If a progress bar records the progress of multiple line items, group these items in adjoining cells and keep the progress bar in-line with the first item it pertains to. For multi-item progress bars that apply to non-consecutive items, consider adding a progress bar in-line with each individual item.
+
+If your table includes multiple progress bars, designate a “Status” or “Progress” column for each one.
+
+
+<img src="./img/progress-bar-in-table.png" alt="A visual example of how to successfully place progress bars in a table." />
+
+<img src="./img/progress-bar-table-example.png" alt="A visual example of how to successfully include multiple progress bars in a table with a 'Status' column designated for each item or task." />
```

Comment:
Would delete this image! Does't really match with the rest of our imagery and is showing an in-product example which I'm not sure is ok to do?

Reply:
I think this image demonstrates the same spacing as the previous one anyway, just with the "Status" column more clearly marked with microcopy. Since we advise using that "Status" column verbally, it shouldn't hurt to remove this image to be safe! 👍